src/styles.scss

```diff
@@ -182,33 +182,72 @@ h1 {
 }
 
 .product-card-list {
-  display: 'flex';
-  flex-flow: row wrap;
-  height: 100%;
+  display: flex;
+  flex-flow: row;
+  width: 90vw;
+  overflow-y: scroll;
 }
 
 .product-card {
+  display: flex;
+  flex-direction: column;
+
   background-color: $container-bg;
-  height: 8rem;
-  width: 16rem;
-  padding: 5px;
+  background: linear-gradient(rgb(245, 245, 245), rgb(224, 224, 224));
+  background-position: center;
+  background-size: 100% 100%;
+
+  height: 20rem;
+  min-width: 11rem;
   margin: 5px;
 
   border: 1px solid black;
   border-radius: 10px;
-  background-color: $container-bg;
 }
 .product-card:hover {
   cursor: pointer;
-  color: $link-btn-color;
+  background-color:rgb(243, 243, 243);
+  filter: drop-shadow(0 0 0.2rem rgba(0, 0, 0, 0.26));
 }
 
 h4 {
   margin: 5px;
 }
 
-.product-card p {
-  margin-left: 5px;
+.card-img {
+  width: 100%;
+  height: 60%;
+  border-bottom: 1px solid black;
+  border-radius: 10px 10px 0px 0px;
+}
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  padding: 3px;
+}
+
+.card-category {
+  margin: 0px;
+  text-transform: uppercase;
```

Comment:
I like how you made them as uppercase using the text-transform! I didn't  even know this exists.

Reply:
:)